util/index.ts

```diff
@@ -0,0 +1,107 @@
+import inquirer, { Answers } from 'inquirer';
+import inquirerFuzzyPath, { FuzzyPathQuestionOptions } from 'inquirer-fuzzy-path';
+
+import { Lang } from '../resources/languages';
+
+import { run as findMissingTranslations } from './find_missing_translations';
+import { default as generateEffectIds } from './gen_effect_id';
+import { run as translateTimeline } from './translate_timeline';
+
+declare module 'inquirer' {
+  interface QuestionMap<T> {
+    fuzzypath: FuzzyPathQuestionOptions<T>;
+  }
+}
+
+const dataFilesMap = {
+  'effect_id.ts': generateEffectIds,
+} as const;
```

Comment:
If you're not going to use the keys as static types, what about this, so you don't need the `answers.choice as keyof typeof dataFilesMap` below?

```suggestion
const dataFilesMap: { readonly [filename: string]: Promise<void> } = {
  'effect_id.ts': generateEffectIds,
};
```